GauchoGrub/app/src/main/java/edu/ucsb/cs/cs190i/gauchogrub/gauchogrub/FavoriteRecyclerAdapter.java

```diff
@@ -0,0 +1,140 @@
+package edu.ucsb.cs.cs190i.gauchogrub.gauchogrub;
```

Comment:
Let's organize our src root folder, it's getting a little crowded with all kinds of files.

└ ui
&nbsp;&nbsp;&nbsp;&nbsp;└ activities
&nbsp;&nbsp;&nbsp;&nbsp;└ fragments
&nbsp;&nbsp;&nbsp;&nbsp;└ adapters